app/services/power_manager/power_status.rb

```diff
@@ -10,6 +10,7 @@ class PowerStatus < Base
 
     def power_state(timeout = DEFAULT_TIMEOUT)
       result = { id: host.id }.merge(HOST_POWER[:na])
+      return random_power_state
```

Comment:
I added this commit to make testing easier. Will remove it once acked. You can comment out this line to switch back to returning the real power status.